### PR TITLE
Add survey response to request details

### PR DESCRIPTION
**Pull request by codella**
## Description

[Team Ultra has been working on exposing a new property in the request show presenter](https://github.com/zendesk/help_center/pull/28939) — `satisfaction_response` — in order to implement the new CSAT details.

This PR aims at augmenting the Copenhagen Theme in order to consume said property, and render the CSAT details as per [the designs in Figma](https://www.figma.com/file/SJX5vix08L1ZOF2Y93vsRL/Customisable-CSAT?type=design&node-id=5607%3A4647&mode=design&t=0i2IWt9B0x8Dfg9X-1).

### Response `absent`
response-absent

### Response `editable`, Rating `absent`
response-editable-rating-absent

### Response `not editable`, Rating `custom text`
response-not-editable-rating-custom-text

### Response `not editable`, Rating `emoji`
response-not-editable-rating-emoji

### Response `not editable`, Rating `numeric`
response-not-editable-rating-numeric

### Response `not editable`, Rating `absent`
rating-not-editable-rating-absent

### Response `editable`, Rating `custom text`
response-editable-rating-custom-text

### Response `editable`, Rating `emoji`
response-editable-rating-emoji

### Response `editable`, Rating `numeric`
response-editable-rating-numeric

### Mobile
mobile

## Checklist

- [x] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [x] :arrow_left: changes are compatible with RTL direction
- [x] :wheelchair: Changes to the UI are [tested for accessibility](./../README.md#accessibility-testing) and compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
- [x] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [x] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: templates/request_page.hbs [hunk 1]
```diff
@@ -128,6 +128,30 @@
...
+                  {{/is}}
+                {{/with}}
+              </div>
+              <div>
+                <a href="{{page_path 'survey_response' survey_response_id=id}}" target="_blank">
+                  {{#if expired}}
+                    {{t 'view_rating'}}
+                  {{else}}
+                    {{t 'edit_rating'}}
+                  {{/if}}
```
Comment (vecerek) [inline templates/request_page.hbs:149]: If the survey response is offered but not expired, we should call `{{t 'add_rating'}}`.
If the survey response is offered and expired, I don't know what to show. Do the designs cover this edge case?

Comment (mancristiana) [reply templates/request_page.hbs:151]: We don't have designs for viewing a response of an expired survey.
The current implementation shows a message informing end-users that their survey has expired: 
Screenshot 2024-01-29 at 14 15 04

Comment (codella) [reply templates/request_page.hbs:151]: We do have them. We skip showing the whole `rating` detail

Screenshot 2024-01-29 at 14 18 33

**Review by Fredx87 — APPROVED** [review]

**Pull request merged**